### DISCUSSION TITLE
post/andrew-tate-stealing-software-revolt

### DISCUSSION THREAD

**Discussion by giscus** [insertish/Portfolio#20] [Comments]
# post/andrew-tate-stealing-software-revolt

Or, "how Andrew Tate is making over $5 million in revenue per month off my software".

https://insrt.uk/post/andrew-tate-stealing-software-revolt

**Comment by insertish** [comment]
![wires revolt](https://user-images.githubusercontent.com/38285861/205592241-52fa0c3d-2f98-4272-9944-227c4c153897.png)

**Comment by ghost** [comment]
Revolt so good that it got stolen

**Comment by ShadowLp174** [comment]
Bruh wtf is wrong with people?!

It is illegal to use and publish software against the license provided, right?

**Reply by Ast3risk-ops** [reply]
You don't see anybody whining about this with ghostscript (also AGPL-licnesed, offers a paid commercial license).

**Reply by alexjyong** [reply]
It's illegal, but it's civil, not criminal.

**Comment by amytimed** [comment]
damn that sucks, they arent being very legal

**Comment by SGtOriginal** [comment]
Rich humans will steal anything. To them it can be quelled with enough money. My respects to you for making this public and hope that you and Revolt come out on top from this, however unlikely it is.

**Comment by ERROR-404-NULL-NOT-FOUND** [comment]
Gotta love how the company doesn't even try to cover it up, they just try to bribe Insert.

**Comment by ghost** [comment]
andrew tate is a stupid fucking

**Comment by JeroenAdam** [comment]
Someone on Twitter bragging TRW now counts 234k members.
Revolt being used on a large scale!
https://twitter.com/T8rabbit/status/1649497553300344844

**Reply by amytimed** [reply]
> Someone on Twitter bragging TRW now counts 234k members. Revolt being used on a large scale! https://twitter.com/T8rabbit/status/1649497553300344844

damn, revolt.chat instance only has 83k

**Reply by JeroenAdam** [reply]
I just visited the same Twitter account, now 170K members, it must be hard times for the Tate's :)

**Comment by ggtylerr** [comment]
Wow, I had no idea this was happening. I hope he isn't giving too much trouble for you guys and he complies (or stops.)

**Comment by searinminecraft** [comment]
this is pretty much one example of license fraud

**Comment by randomairborne** [comment]
Have you contacted the FSF about this violation?

**Reply by Ast3risk-ops** [reply]
Maybe the EFF could help, though they're US-based.

**Comment by ghost** [comment]


**Comment by Ast3risk-ops** [comment]
You probably won't be able to do anything because Dr. Eggman Tate is too powerful.

**Comment by src-ry** [comment]
shameful

**Comment by bonio350** [comment]
seems like you couldn't take it down. rekt

**Comment by algj** [comment]
https://youtu.be/xR5d4Ba4FZg

**Reply by searinminecraft** [reply]
Looks like the frontend has been leaked, but if the backend and other revolt components (january, etc.) also leaked then theyre "technically" AGPLv3 compliant :trollface:

**Comment by alicegn-bolorg** [comment]
You can fill in here: https://www.gnu.org/licenses/gpl-violation.html

**Comment by ghost** [comment]
Their slogan: Escape slavery to be my slave
